dexbot/strategies/relative_orders.py

```diff
@@ -52,19 +74,46 @@ def __init__(self, *args, **kwargs):
         self.manual_offset = self.worker.get('manual_offset', 0) / 100
         self.order_size = float(self.worker.get('amount', 1))
         self.spread = self.worker.get('spread') / 100
+        self.is_reset_on_partial_fill = self.worker.get('reset_on_partial_fill', True)
+        self.partial_fill_threshold = self.worker.get('partial_fill_threshold', 30) / 100
+        self.is_reset_on_market_trade = self.worker.get('reset_on_market_trade', False)
+        self.is_reset_on_price_change = self.worker.get('reset_on_price_change', False)
+        self.price_change_threshold = self.worker.get('price_change_threshold', 2) / 100
+        self.is_custom_expiration = self.worker.get('custom_expiration', False)
+
+        if self.is_custom_expiration:
+            self.expiration = self.worker.get('expiration_time', self.expiration)
+
+        self.last_check = datetime.now()
+        self.min_check_interval = 8
 
         self.buy_price = None
         self.sell_price = None
 
         self.initial_balance = self['initial_balance'] or 0
         self.worker_name = kwargs.get('name')
         self.view = kwargs.get('view')
-        self.check_orders()
+
+        # Check for conflicting settings
+        if self.is_reset_on_price_change and not self.is_center_price_dynamic:
+            self.log.error('reset_on_price_change requires Dynamic Center Price')
```

Comment:
Please use the human readable format for the names

Reply:
@mikakoi do you mean human readable format inside the log message?

Reply:
Yes, Just change the message to `'Reset orders on center price change' requires 'Dynamic center price'` or something